utils/helpers.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env node */
+'use strict';
+
+const leadingWhiteSpace = /^[ \t\r\n]+/;
+const trailingWhiteSpace = /[ \t\r\n]+$/;
+const WHITESPACE = /^[ \t\r\n]+$/;
+
+const isWhitespaceTextNode = function(node) {
+  return node && node.type === 'TextNode' && WHITESPACE.test(node.chars);
+};
+
+const hasLeadingOrTrailingWhiteSpace = function(chars) {
+  return leadingWhiteSpace.test(chars) || trailingWhiteSpace.test(chars);
+};
+
+const stripWhiteSpace = function(chars) {
+  /*
+    Replacing multiple ' ', '\n' and '\t'(leading/trailing) into a single whitespace.
+  */
+  chars = chars || '';
+  return chars.replace(leadingWhiteSpace, ' ').replace(trailingWhiteSpace, ' ');
+};
+
+const stripNoMinifyBlocks = function(nodes) {
```

Comment:
why did you not use the normal function syntax here? (e.g. `function stripNoMinifyBlocks(nodes) { ... }` instead of the `const stripNoMinifyBlocks = function(nodes) { ... }`)

Reply:
No specific reason, since `stripNoMinifyBlocks` is never going to change. I have used `const`. Shall I use the normal syntax here?

Reply:
yeah, I think that would be best, thanks